foundations/javascript_basics/understanding_errors.md

```diff
@@ -130,8 +130,8 @@ At this point, you might be wondering how we can resolve these errors.
 
 1. We can start by understanding that the error message is your friend - not your enemy. Error messages tell you *exactly* what is wrong with your code, and which lines to examine to find the source of the error. Without error messages it would be a *nightmare* to debug our code - because it would still not work, we just wouldn't know why!
 1. Now, its time to Google the error! Chances are, you can find a fix or explanation on StackOverflow or in the documentation. If nothing else, you will receive more clarity as to why you are receiving this error.
-1. Use the debugger! As previously mentioned, the debugger is great for more involved troubleshooting, and is a critical tool for a developer. You can set breakpoints, view the value of any given variable at any point in your application’s execution, step through code line by line, and more! It is an extremely valuable tool and every programmer should know how to use it. [This tutorial dives into the Chrome Debugger](https://developer.chrome.com/docs/devtools/javascript/). Warning: In point 4 of step 3 of the tutorial, devtools will pause on the second line rather than at the declaration of the function. Don't worry this is expected.
-1. Make use of the console! `console.log()` is a popular choice for quick debugging. For more involved troubleshooting, using the debugger might be more appropriate, but using `console.log()` is great for getting immediate feedback without needing to step through your functions. There are also other useful methods such as `console.table()`, `console.trace()`, and more! You can find additional methods [here](https://www.w3schools.com/jsref/obj_console.asp).
+1. Use the debugger! As previously mentioned, the debugger is great for more involved troubleshooting, and is a critical tool for a developer. You can set breakpoints, view the value of any given variable at any point in your application’s execution, step through code line by line, and more! It is an extremely valuable tool and every programmer should know how to use it. This tutorial dives into the [Chrome Debugger](https://developer.chrome.com/docs/devtools/javascript/). Warning: In point 4 of step 3 of the tutorial, devtools will pause on the second line rather than at the declaration of the function. Don't worry this is expected.
+1. Make use of the console! `console.log()` is a popular choice for quick debugging. For more involved troubleshooting, using the debugger might be more appropriate, but using `console.log()` is great for getting immediate feedback without needing to step through your functions. There are also other useful methods such as `console.table()`, `console.trace()`, and more! Read the W3school's article to find additional [methods](https://www.w3schools.com/jsref/obj_console.asp) of the `console` object.
```

Comment:
Similarly to the above, could the `Read the W3school's article to find additional [methods](https://www.w3schools.com/jsref/obj_console.asp) of the `console` object.` be moved to the additional resources please?

The wording will need to be change, as it will need a more descriptive name than `methods`. My suggestion would be:

```markdown
- Learn about the [window console object](https://www.w3schools.com/jsref/obj_console.asp).
```

Reply:
what about i just replace the `methods` with `window console object`? like this:
Read the W3schools article to find additional [window console object's](https://www.w3schools.com/jsref/obj_console.asp) methods. Just suggesting, either way, i do not mind following your suggestion instead.

Reply:
`[window console object methods]` (no `'s`) would be perfectly fine :+1:

Reply:
We'll still need the window console but moved to additional resources

Reply:
Gah. This is embarrassing. I forgot to put it there :/. Let me boot up my PC for a sec; I will fix it.

Reply:
Eh, no worries. I've had my fair share too :P